### ISSUE TITLE
Add diversions information to active line and variant.

### ISSUE THREAD

**Issue by psimonek** [psimonek/mhd-brno#17]
Asked for API, REST, RSS ??? accessibility. Cannot find public source. It looks it is private connection only. Waiting for the answer.

**Comment by psimonek** [comment]
Request sent to DPMB - they promised to give access to API. Checking server (PHP) / client (JS) requirement to work with their API.
